waspc/data/Generator/templates/sdk/wasp/server/config.ts

```diff
@@ -1,9 +1,9 @@
 {{={= =}=}}
```

Comment:
I've started to work on enabling strict null checks while working on env variables, so these are some changes I left from that effort - `env` -> `nodeEnv` since I imagined that the import of env vars would look like `import { env } from 'wasp/server'`